File: ts-sdk/src/sdk/models/operations/getapipages.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetApiPagesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata({ elemType: shared.Page })
  pages?: shared.Page[];

  @SpeakeasyMetadata()
  statusCode: number;
}

File: python-sdk/src/foremapi/models/operations/get_api_pages.py
import dataclasses
from typing import Optional
from ..shared import page as shared_page


@dataclasses.dataclass
class GetAPIPagesResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    pages: Optional[list[shared_page.Page]] = dataclasses.field(default=None)
    
